pkg/daemon/handler_linux.go

```diff
@@ -57,10 +60,27 @@ func createShortSharedDir(pod *v1.Pod, volumeName string) (err error) {
 	return err
 }
 
+func makeCleanLog(logStr string) string {
+	if logStr == "" {
+		return "empty string"
+	}
+	cleanLog := strings.Replace(logStr, "\n", "", -1)
+	cleanLog = strings.Replace(logStr, "\r", "", -1)
+	return cleanLog
+}
+
 func removeShortSharedDir(pod *v1.Pod, volumeName string) (err error) {
 	sharedDir := getShortSharedDir(pod.UID, volumeName)
 	if _, err = os.Stat(sharedDir); os.IsNotExist(err) {
-		klog.Errorf("shared directory %s does not exist to unmount, %s", sharedDir, err)
+		dir := makeCleanLog(sharedDir)
+		klog.Errorf("shared directory %s does not exist to unmount, %s", dir, err)
```

Comment:
We can make `sharedDir` double-quoted in the logs to provide accurate information.